docker-build-win64.sh

```diff
@@ -588,6 +588,9 @@ ffversion="$(dpkg-parsechangelog --show-field Version)"
 if [[ -f "patches/series" ]]; then
     quilt push -a
 fi
+# Workaround for GCC bug causing misaligned AVX instructions
+CFLAGS+=" -Wa,-muse-unaligned-vector-move"
+CXXFLAGS+=" -Wa,-muse-unaligned-vector-move"
```

Comment:
How about we move this mingw gcc flag to https://github.com/FFmpeg/FFmpeg/blob/b818dff8d8e16f0831a379a5cdc15e813474a841/configure#L5795

```
# Workaround for GCC bug causing misaligned AVX instructions
enabled x86_64 && check_cflags -Wa,-muse-unaligned-vector-move
enabled x86_64 && check_cxxflags -Wa,-muse-unaligned-vector-move
```

Reply:
I'd rather not touch the configure script. It is much more harder to maintain than our CI script

Reply:
I agree. But we should also make the patch self-contained so that users can get reproducible results without using our build scripts, otherwise users will need to read our build scripts to realize the quirks exist.